api/room_resource/schema.py

```diff
@@ -290,6 +297,21 @@ def resolve_get_resources_by_room_id(self, info, room_id):
             resources.append(room_resource)
         return RoomResources(roomResources=resources)
 
+    @Auth.user_roles('Admin')
+    def resolve_resource_by_name(self, info, search_name):
+        resource = []
+        resource_name = search_name.lower().replace(" ", "")
+        if not resource_name:
+            raise GraphQLError("Please input Resource Name")
+        query = Resource.get_query(info).filter_by(state="active")
```

Comment:
What does the query represent? Use variable names that reflect the value being assigned to them